macros/parsers/parserPopUp.pl

```diff
@@ -15,82 +15,164 @@
-by default, but can be customized with a C<placeholder> option.
+Note that drop-down menus can not contain mathematical notation, only
+plain text. This is because the browser's native menus are used, and
+these can contain only text, not mathematics or graphics.
 
-C<DropDownTF()> is like C<DropDown> with options being localized versions of
-"True" and "False". 1 is understood as "True" and 0 as "False". The initial
-letter of the localized word is understood as that word if those letter are
-different. All of this is case-insensitive. Also, in static output (PDF, PTX)
-C<showInStatic> is 0. It is assumed that context makes the menu redundant.
+The difference between C<PopUp()> and C<DropDown() >is that in HTML,
+the latter will have an unselectable placeholder value.  This value
+is '?' by default, but can be customized with a C<placeholder> option.
+
+C<DropDownTF()> is like C<DropDown> with options being localized
+versions of "True" and "False". 1 is understood as "True" and 0 as
+"False". The initial letter of the localized word is understood as
+that word if those letter are different. All of this is not case
+sensitive. Also, in static output (PDF, PTX) C<showInStatic> defaults
+to 0. It is assumed that text preceding the drop-down makes the menu
+redundant.
+
+The entries in the choices array can either be the actual strings to
+be used in the drop-down menu (which is known as a "label" for the
+option input in HTML) or C<< { label => value } >> where C<label> is
+the text string to display in the drop-down list and C<value> is the
+value to for the option input for this choice. The "value" is what is
+actually submitted when a student submits an answer, and this is what
+will appear in the past answers table, feedback messages, etc. If an
+option is not set as a hash in this way, the text of the option serves
+as both the label and the value.
 
 By default, the choices are left in the order that you provide them,
 but you can cause some or all of them to be ordered randomly by
 enclosing those that should be randomized within a second set of
 brackets.  For example
 
-    $radio = PopUp([
-                     "First Item",
-                     ["Random 1","Random 2","Random 3"],
-                     "Last Item"
-                   ],
-                   "Random 3"
-                 );
+    $dropdown = DropDown(
+        [
+            "First Item",
+            [ "Random 1", "Random 2", "Random 3" ],
+            "Last Item"
+        ],
+        "Random 3"
+    );
 
-will make a pop-up menu that has the first item always on top, the
-next three ordered randomly, and the last item always on the bottom.
-In this example
+will make a list of options that has the first item always on top,
+the next three ordered randomly, and the last item always on the
+bottom.  In this example
 
-    $radio = PopUp([["Random 1","Random 2","Random 3"]],2);
+    $dropdown = DropDown([["Random 1","Random 2","Random 3"]],2);
```

Comment:
This could use the clean up as well.  `perltidy` gives
```suggestion
    $dropdown = DropDown([ [ "Random 1", "Random 2", "Random 3" ] ], 2);
```